features/ptp/README.md

```diff
@@ -11,11 +11,57 @@
 * [OCP4 PTP Operator](https://github.com/openshift/ptp-operator)
 * [ptp4l grand master down with error timed out while polling for tx timestamp](https://access.redhat.com/solutions/2107131)
 
-## TL;DR
+## Instructions
 
-* Create a `ptpconfig.yaml` that fits your environment
+* Create a `myvars` file that fits your environment. You can use the [myvars.example](myvars.example) as inspiration.
 * Run the `deploy.sh` script
 
-It will deploy the PTP operator and select one of the masters randomly to act as the grandmaster and the other nodes as slaves.
+## Testing
```

Comment:
I'd say to rename this section as 'Deploying unreleased versions' instead and just explain the secret and the catalog source.